hw/ip/otbn/doc/_index.md

```diff
@@ -587,8 +587,7 @@ A fatal alert can only be cleared by resetting OTBN through the `rst_ni` line.
 ### Reaction to Life Cycle Escalation Requests
 
 OTBN receives and reacts to escalation signals from the [life cycle controller]({{< relref "/hw/ip/lc_ctrl/doc#security-escalation" >}}).
-
-An escalation request signaled through the `lc_escalate_en_i` signal results in the same action as a [fatal error](#design-details-fatal-errors) but does not raise a fatal alert.
+An incoming life cycle escalation is a fatal error of type `lifecycle_escalation` and treated as described in the section [Fatal Errors](#design-details-fatal-errors).
```

Comment:
May be worth clarifying this only happens if the life cycle escalation occur whilst OTBN is running?

> Note a fatal error only occurs if the life cycle escalation happens whilst OTBN is running.

Reply:
No, a fatal error occurs in any case. Only the handling differs depending on the state OTBN is in, as discussed in the linked section of the docs.

Can you reread the docs and let me know where we could improve things to make this clear?

Reply:
Ah ok, I had presumed that setting `ERR_BITS` always occurred on a fatal error, however the linked section makes it clear that only happens if OTBN is busy at the time. So that should suffice for what I was trying to express.